cmd/rekor-server/app/root.go

```diff
@@ -60,6 +61,8 @@ func init() {
 	rootCmd.PersistentFlags().String("trillian_log_server.address", "127.0.0.1", "Trillian log server address")
 	rootCmd.PersistentFlags().Uint16("trillian_log_server.port", 8090, "Trillian log server port")
 	rootCmd.PersistentFlags().Uint("trillian_log_server.tlog_id", 0, "Trillian tree id")
+	rootCmd.PersistentFlags().Var(&logRangeMap, "trillian_log_server.log_id_ranges", "list of tree ids and ranges")
```

Comment:
Is there any need to specify a different log server IP/port for each tree ID here, or is a single trillian instance sufficient for multiple trees?

Reply:
I could see us needing to eventually, but can't think of a real reason yet. WDYT? One trillian instance can handle many trees.